XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,573 @@
+
+#### 2.1.1. Object ID
+
+The ID of this object will be a hash that incorporates the `Subject`, `Issuer`, and `CredentialType` fields, combined with a unique space key for `Credential` objects, which will be defined during implementation.
+
+#### 2.1.2. `Flags`
+| Flag Name | Flag Value |
+|-----------|------------|
+|`lsfAccepted`|`0x00010000`|
+
+`lsfAccepted` is on if the subject of the credential has accepted the credential. If it is off, the issuer has the reserve burden; if it is on, the burden moves to the subject.
+
+#### 2.1.3. `CredentialType`
+
+This value is similar to the NFT `Taxon` value, where the value's meaning will be decided by the issuer. It may be the same as a claim in a VC, but could also represent a subset of such a claim.
+
+### 2.2. Account Deletion
+
+The `Credential` object is not a [deletion blocker](https://xrpl.org/docs/concepts/accounts/deleting-accounts/#requirements).
+
+## 3. On-Ledger Object: `DepositPreauth`
+
+The `DepositPreauth` object tracks a preauthorization from one account to another. This object already exists on the XRPL, but is being extended as a part of this spec to also support credential preauthorization.
+
+### 3.1. Fields
+<details>
+<summary>
+
+As a reference, [here](https://xrpl.org/docs/references/protocol/ledger-data/ledger-entry-types/depositpreauth/#depositpreauth-fields) are the existing fields for the `DepositPreauth` object.
+</summary>
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`Account`|✔️|`string`|`AccountID`|The account that granted the preauthorization (the destination of the preauthorized payments).|
+|`Authorize`|✔️|`string`|`AccountID`|The account that received the preauthorization (the sender of the preauthorized payments).|
+|`LedgerEntryType`|✔️|`string`|`UInt16`|The value `0x0070`, mapped to the string `"DepositPreauth"`, indicates that this is a `DepositPreauth` object.|
+|`OwnerNode`|✔️|`string`|`UInt64`|A hint indicating which page of the sender's owner directory links to this object, in case the directory consists of multiple pages. Note: The object does not contain a direct link to the owner directory containing it, since that value can be derived from the `Account.PreviousTxnID`.|
+|`PreviousTxnID`|✔️|`string`|`Hash256`|The identifying hash of the transaction that most recently modified this object.|
+|`PreviousTxnLgrSeqNumber`|✔️|`number`|`UInt32`|The index of the ledger that contains the transaction that most recently modified this object.|
+</details>
+
+We propose these modifications:
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`Authorize`| |`string`|`AccountID`|The account that received the preauthorization. (The sender of the preauthorized payments.)|
```

Comment:
I would explicitly state _here_ that this field is the same but becomes optional.